### ISSUE TITLE
No $token

### ISSUE THREAD

**Issue by stefan94antonic** [tymondesigns/jwt-auth#1478]
### Subject of the issue
JWTAuth:attempt($credentials) returns nothing; not true, not false.

### Your environment
| Q                 | A
| ----------------- | ---
| Bug?              |  yes
| New Feature?      | no 
| Framework         | Laravel 
| Framework version | 5.5
| Package version   | 1.0.0-rc.1
| PHP version       | 7

**Comment by leon0399** [comment]
THis methond should return token, not a bool

UPD: Read - https://jwt-auth.readthedocs.io/en/develop/auth-guard/#attempt

**Comment by 604bpierce** [comment]
I'm having this same problem. I'm getting back a bool (true) instead of a token on the `auth()->attempt([$credentials]);` request.

_Edit: using `auth('api')->attempt($credentials)` seems to solve this issue_

**Comment by markuskoehler** [comment]
@604bpierce yeah also found that, this happens if you don't set your default guard in `config/auth.php` to `'api'`. If you do, it is perfectly fine with just using `auth()` instead of `Auth::guard('api')` and `auth()->attempt()` will always return a token instead of bool

**Comment by GonzaloGPF** [comment]
I had same issue. I was facing this problem because I was using `auth()->attempt($credentials, $remember)`.
If you use the second argument ($remember) it will always use your **web** auth driver, despite of your defaults in **config/auth.php**. So, you must use `auth()->attempt($credentials)`.
